public/doc/swagger-2.yaml

```diff
@@ -12,8 +12,8 @@ host: 'localhost:3000'
 tags:
   - name: admins
     description: Admin-only APIs
-  - name: approvers
-    description: Approvers APIs
+  - name: users
```

Comment:
You can't have two `users` tags. Should the regular user be `requesters` or `guests`?